src/patchRequire.js

```diff
@@ -196,10 +197,21 @@ export default function patchRequire(vol, unixifyPaths = false, Module = require
             // Don't search further if path doesn't exist
             const curPath = paths[i];
             if (curPath && stat(curPath) < 1) continue;
-            var basePath = correctPath( path.resolve(curPath, request) );
-            var filename;
-
+            var basePath = path.resolve(curPath, request);
             var rc = stat(basePath);
+
+            // check if this is a windows paths and if it should be corrected
+            if( rc < 0 && isWindowsPath(basePath) ) {
```

Comment:
this block is skipped if the uncorrected base path is valid, or if the path isn't a windows-style path anyway (in which case, it's not likely going to need correction)